docker/Dockerfile.gpu

```diff
@@ -0,0 +1,70 @@
+FROM nvidia/cuda:9.0-base-ubuntu16.04
```

Comment:
Should this be `FROM nvidia/cuda:9.1-cudnn7-devel-ubuntu16.04`?

Reply:
got it